src/commander.js

```diff
@@ -72,30 +68,96 @@ define([
         cmd.input.val("").removeClass("alert-danger");
     }
 
-    function setupAutocomplete(cmd, choices) {
-        var input = cmd.input;
-        input.atwho({
-            at: "",
-            data: choices,
-            limit: 50,
-            maxLen: Infinity,
-            suffix: " ",
-            tabSelectsMatch: true,
-            callbacks: {
-                // match first argument only
-                matcher: function (flag, subtext) { return subtext; }
-            }
-        });
+    function setupAutocomplete(cmd) {
+        if (cmd.atwhoConfig) {
+            return;
+        }
+        var typeNames = _.pluck(fn.getQuestionMap(cmd.vellum), "typeName"),
+            positions = ["after", "before", "in", "first in"],
```

Comment:
I love that there's an autocomplete for these; it makes the commander much easier to use.